terragrunt/modules/release-distribution/lambdas/doc-router/index.js

```diff
@@ -66,10 +66,17 @@ exports.handler = (event, context, callback) => {
         return temp_redirect('https://www.rust-lang.org/learn', callback);
     }
 
-    // Forward versioned documentation as-is.
-    if (/^\/\d/.test(request.uri)) {
+    // Forward patch versioned documentation as-is.
+    if (/^(\d+)\.(\d+).(\d+)\/(.*)/.test(request.uri)) {
         return callback(null, request);
     }
+    // Include patch version 0 if only minor version is provided
+    // Todo("Forward to latest patch version if possible")
+    const minor_versioned_pattern = /^(\d+)\.(\d+)\/(.*)/
```

Comment:
This regex looks good 👍
![image](https://github.com/user-attachments/assets/5a45edbf-69cb-4fec-a03f-3bb755716005)